### PR TITLE
new-features-user

### PR DESCRIPTION
**Pull request by ArsalanSavand**
I've added  2 components:

1. Settings component : which will update current user's settings. ( name, location, etc)

2.Profile component : which will display current user's details.

@amirm3hdi I will leave the rest of designing and frontEnd to you.

@Alir3z4 hope you enjoyed it.

### CODE REVIEW

File: app/entry_new/entry_new_controller.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     * newPost
+     *
+     * @method newPost
+     * @desc Submit newPost form
+     *
+     * @param form {object} Form data to submit
+     */
+    $scope.newPost = function(form) {
+        form.loading = true;
+        form.site = "8ee0840d-f95b-4d94-aa78-c2262bb1908e"; // Debug
```
Comment (Alir3z4) [inline app/entry_new/entry_new_controller.js]: @amirm3hdi @ArsalanSavand This should not be hardcoded.

File: app/entry_new/entry_new_controller.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        NewEntryService.create(form).then(
+            function(data, status, headers, config) {
+                // Success message
+                $mdToast.showSimple("Entry successfully added !");
```
Comment (Alir3z4) [inline app/entry_new/entry_new_controller.js]: You can say just "Entry Added"

**Comment by Alir3z4** [comment]
@ArsalanSavand I've submitted 2 comments, please take care of them then let me know when you're done.

**Comment by Alir3z4** [comment]
lemme know when this is ready so I can have a look again.

**Comment by amirm3hdi** [comment]
@Alir3z4 All components work currently,
I've implemented the user-site-in-url method and is compatible with all other components.

For now we don't have design update of new components,
Correct documentations for functions/controllers
And there's still room for improvement

Anyway, You can **Merge** this branch now.

**Pull request merged**